students/john_navitsky/session06/test_mailroom.py

```diff
@@ -32,12 +34,206 @@ def test_mock_input():
     assert mailroom.safe_input(mock=True,mock_in="foo") == "foo"
 
 
-# def test_print_thank_you():
-#      # verify the print_thank_you() function
-#     out = io.StringIO()
-#     mailroom.print_thank_you(0,"testfull",out)
-#     output = out.getvalue()
-#     out.close()
-#     assert "Dearest Joe Smith," in output
+def test_print_thank_you():
+    # verify the print_thank_you() function
+    a_donor = {
```

Comment:
good use of donor example in test.

but shouldn't there be a donations entry too?

Reply:
I probably overlooked it in the lesson description.

Reply:
You’ve got it elsewhere ....

Reply:
You’ve got it elsewhere ....